internal/provider/resource_build.go

```diff
@@ -39,6 +40,14 @@ type BuildResourceModel struct {
 	SBOMs types.Map `tfsdk:"sboms"`
```

Comment:
I'd love to replace this with a structured type.  LMK if you have ideas @imjasonh because I've hit problems trying to use a structured type here.